recipe/meta.yaml

```
@@ -8,17 +8,18 @@ package:
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pynng-{{ version }}.tar.gz
   sha256: 5edd3ccbeda2264d750d81533a11d45b5efbf8d99da65ba1f4569636f0eb9b70
+  patches:
+    - fix-windows-builds.patch
```

```suggestion
    - fix-windows-builds.patch  # [win]
```
Maybe it helps to apply the patch only for the windows build

---

Thank you for the review @jan-janssen!

I don't think this will help, the patch only affects build settings on windows...